index.html

```diff
@@ -138,7 +138,24 @@ <h3><span>languages</span><img src="graphics/arr-down.svg" alt=""></h3>
                 <li><img src="graphics/css.svg" alt="">&nbsp;&nbsp;&nbsp;<span>CSS</span></li>
             </ul>
             <h3><span>frameworks</span><img src="graphics/arr-right.svg" alt=""></h3>
-            <h3><span>skills</span><img src="graphics/arr-right.svg" alt=""></h3>
+            <h3 class="skills"><span>skills</span><img src="graphics/arr-right.svg" alt=""></h3>
+        </div>
+    </section>
+    <section class="contact">
+        <div class="contact-container">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d
+            love to help with it </p>
+            <form action="https://formspree.io/f/mwkywlgp" method="post">
+                <input type="text" id="name" name="user_name"
+                placeholder="John Doe" maxlength="30" required>
+                <input type="email" id="mail" name="user_email"
+                placeholder="john.doe@example.com" required>
+                <textarea id="msg" name="user_message"
+                placeholder="Write your message here" maxlength="500" required></textarea>
+                <button type="submit">Get in touch</button>
+            </form>
```

Comment:
- [ ] [OPTIONAL] Please make sure to use some spacing between the form's elements to match the Figma design.